### PR TITLE
ENDOC-729-MS-resource

### PR DESCRIPTION
**Pull request by jyunmitch**
This is a short update but the Eng tickets were rather hard to pull together so I'm asking all of your review. This is to document the healthCheckIngress and ingressPath for the miscroservices. 

The recommendation for the API claim in the v1 to v5 conversion is included in another ticket.

### CODE REVIEW

File: vuepress/docs/next/docs/curate/bundle-details.md [hunk 1]
```diff
@@ -94,22 +94,24 @@ The following is a list of specifications for the bundle descriptor and its comp
...
 |`permissions`|[Permission[]](#permission-specification)|No| | List of permissions to grant to the microservice |
 |`roles`|String[]|No||Exposed security roles|
 |`env`|[EnvironmentVariable[]](#environmentvariables-specification)|No||Required environment variables|
 | |[Microservices Environment Variables](#microservices-environment-variables)|No||Entando-provided env variables for MS |
 |`commands`|[Command[]](#command-specification)|No||Custom command(s) definitions|
 |`version`|String|Required only for a custom stack||Microservice version override|
-> ^Note: Oracle and other DBMS types are not supported for automatic deployment in a container. Bundle env variables should be used instead, similar to connecting the EntandoApp to an [external database](../../tutorials/devops/external-db.md).
+> ^ dbms none: Oracle and other DBMS types are not supported for automatic deployment in a container. Bundle env variables should be used instead, similar to connecting the EntandoApp to an [external database](../../tutorials/devops/external-db.md).
 
+>^^ ingressPath: Customizing the ingress path for the health check requires an additional rewrite annotation or an API claim for the bundle.
```
Comment (jyunmitch) [inline vuepress/docs/next/docs/curate/bundle-details.md:114]: @nshaw, this seems unsatisfactory. It didn't sound better to say the path works only partially for v5 bundles. If it's going to change soon, then I won't worry but if not, can I get more information about the workaround so we can add it to another section or something?

File: vuepress/docs/next/docs/curate/bundle-details.md [hunk 2]
```diff
@@ -94,22 +94,24 @@ The following is a list of specifications for the bundle descriptor and its comp
...
 ### Microservices Specifications
-|Name|Type|Required|Possible Values|Description|
+|Name|Type|Required|Possible Values [default]|Description|
 |:-|:-|:-|:-|:------------------------|
 |`name`|String|Yes||Microservice name|
 |`stack`|Enum|Yes|*spring-boot<br/>*node<br/>*custom|Microservice stack |
 |`dbms`|Enum|No|*none^  *embedded  *postgresql  *mysql  |DBMS required by the MS to provide services|
-|`ingressPath`|String|No||Custom ingress path|
 |`healthCheckPath`|String|No||Endpoint for a health check|
+|`healthCheckIngress`|Enum| Yes |*[canonical]  *custom| Use the canonical address or customize with `ingressPath`
```
Comment (nshaw) [inline vuepress/docs/next/docs/curate/bundle-details.md:103]: I thought this change was backed out for 7.3. @firegloves am I correct?

File: vuepress/docs/next/docs/curate/bundle-details.md (hunk 1 above)
Comment (nshaw) [reply vuepress/docs/next/docs/curate/bundle-details.md:114]: @jyunmitch I would drop this line. We have a bug ticket for this but explaining the overall situation is a bit complicated. What we have works if you adjust the ingressPath - but it doesn't let you control the full path and that's the new feature/bug.

**Review by nshaw — CHANGES_REQUESTED** [review]

File: vuepress/docs/next/docs/curate/bundle-details.md (hunk 2 above)
Comment (firegloves) [reply vuepress/docs/next/docs/curate/bundle-details.md:103]: Yes @nshaw @jyunmitch, we have reverted the change, so it is no longer implemented

Comment (nshaw) [reply vuepress/docs/next/docs/curate/bundle-details.md:103]: @firegloves thanks!

Comment (jyunmitch) [reply vuepress/docs/next/docs/curate/bundle-details.md:103]: @nshaw, I'll just drop this PR altogether. that will keep the ingressPath as it was.

File: vuepress/docs/next/docs/curate/bundle-details.md [hunk 3]
```diff
@@ -94,22 +94,24 @@ The following is a list of specifications for the bundle descriptor and its comp
...
+|`ingressPath`^^|String|No||Custom ingress path for health check|
 |`deploymentBaseName`|String|No||Used to define custom pod names|
 |`permissions`|[Permission[]](#permission-specification)|No| | List of permissions to grant to the microservice |
 |`roles`|String[]|No||Exposed security roles|
 |`env`|[EnvironmentVariable[]](#environmentvariables-specification)|No||Required environment variables|
 | |[Microservices Environment Variables](#microservices-environment-variables)|No||Entando-provided env variables for MS |
 |`commands`|[Command[]](#command-specification)|No||Custom command(s) definitions|
 |`version`|String|Required only for a custom stack||Microservice version override|
-> ^Note: Oracle and other DBMS types are not supported for automatic deployment in a container. Bundle env variables should be used instead, similar to connecting the EntandoApp to an [external database](../../tutorials/devops/external-db.md).
+> ^ dbms none: Oracle and other DBMS types are not supported for automatic deployment in a container. Bundle env variables should be used instead, similar to connecting the EntandoApp to an [external database](../../tutorials/devops/external-db.md).
```
Comment (nshaw) [inline vuepress/docs/next/docs/curate/bundle-details.md:112]: @jyunmitch This language was useful,  I thought.

File: vuepress/docs/next/docs/curate/bundle-details.md (hunk 2 above)
Comment (nshaw) [reply vuepress/docs/next/docs/curate/bundle-details.md:103]: @jyunmitch The language around oracle and external dbs was still useful but it's a pretty small change in the end.

**Review by nshaw — APPROVED** [review]

**Pull request merged**